apps/condo/domains/ticket/tasks/sendTicketCommentCreatedNotifications.js

```diff
@@ -100,7 +101,8 @@ const sendTicketCommentCreatedNotifications = async (commentId, ticketId) => {
                         commentContent: createdComment.content || EMPTY_CONTENT,
                         commentType: createdComment.type,
                         commentTypeMessage: CommentTypeMessage,
-                        commentCreatedAt: dayjs(createdComment.createdAt).format('YYYY-MM-DD HH:mm'),
+                        // TODO(DOMA-9568): use user timezone after adding it
+                        commentCreatedAt: dayjs(createdComment.createdAt).tz(DEFAULT_ORGANIZATION_TIMEZONE).format('YYYY-MM-DD HH:mm'),
```

Comment:
DEFAULT_TIMEZONE

Reply:
This variable is used throughout the project

I think this pr is not the best place for this change)